shub/image/list.py

```diff
@@ -81,7 +81,10 @@ def list_cmd(image_name, project, endpoint, apikey):
             click.echo(logs)
             raise ShubException('Container with list cmd exited with code %s' % exit_code)
         logs = logs.decode('utf-8') if isinstance(logs, binary_type) else logs
-        return {'spiders': utils.valid_spiders(logs.splitlines())}
+        return {
+            'project_type': 'scrapy',
```

Comment:
We haven't released non-scrapy deploys yet, so probably it's fine to assume `scrapy` project type if `shub-image-info` wasn't found.